main.tf

```diff
@@ -20,7 +20,7 @@ data "template_cloudinit_config" "this" {
 resource "aws_autoscaling_group" "this" {
   name_prefix               = var.name
   desired_capacity          = var.asg_desired_capacity
-  health_check_grace_period = 300
+  health_check_grace_period = 600
```

Comment:
10 minutes of grace on the health check is a long damn time. Does it really take that long to start up?

Reply:
It takes more than 5 minutes sometimes as it does pre-flight checks on every repo.